CHANGES.md

```diff
@@ -1,5 +1,94 @@
+- You can now assign `AutomationConditions` to asset checks, via the `automation_condition` parameter on `@asset_check` or `AssetCheckSpec`.
+- You can now assign `AutomationConditions` to observable source assets, via the `automation_condition` parameter on `@observable_source_asset`.
+- [experimental] You can now define custom subclasses of `AutomationCondition` to execute arbitrary Python code in the context of a broader expression. This allows you to compose built-in conditions with custom business logic.
+- The `target` arguments on schedules and sensors are now marked stable, allowing a stable way for schedules and sensors to target asset selections without needing to define a job.
+
+### Integrations
+
+- Introduced a slate of integrations with business intelligence (BI) tools, enabling dashboards, views, and reports to be represented in the Dagster asset graph.
+  - Power BI [docs]
+  - Looker [docs]
+  - Tableau [docs]
+  - Sigma [docs]
+- Pipes 1.9 changes: TODO
+- Rich asset metadata in ELT integrations: TODO
+
+### Airlift
+
+- TODO
+
+### UI
+
+- Several changes have been made to the information architecture to make it easier to find what you’re looking for:
+  - Backfills have been moved from their own tab underneath the Overview page to entries within the table on the Runs page. This reflects the fact that backfills and runs are similar entities that share most properties. You can continue to use the legacy Runs page with the “Revert to legacy Runs page” user setting. ([GitHub Discussion](https://github.com/dagster-io/dagster/discussions/24898))
+  - “Jobs” is now a page reachable from the top-level navigation pane. It replaces the Jobs tab within the Overview page.
+  - “Automations” is now a page reachable from the top-level navigation pane. It replaces the schedule and sensor tabs within the Overview page.
+- `@asset` and `AssetSpec` now have a `kinds` attribute that enables specifying labels that show up on asset nodes in the asset graph in the UI. This supersedes the `compute_kind` attribute.
+
+## Changes since 1.8.13 (core) / 0.24.13 (libraries)
+
+### New
+
+- The `tags` parameter to `@asset` and `AssetSpec` is no longer marked as experimental.
+- The `@observable_source_asset` decorator now supports an `automation_condition` argument.
+- `AutomationCondition` and associated APIs are no longer marked as experimental.
+- Added a new `use_user_code_server` parameter to `AutomationConditionSensorDefinition`. If set, the sensor will be evaluated in the user code server (as traditional sensors are), allowing custom `AutomationCondition` subclasses to be evaluated.
+- Added a new column to the BulkActions table and a new BackfillTags table to improve performance when filtering Backfills. To take advantage of these performance improvements, run `dagster instance migrate`. This migration involves a schema migration to add the new column and table, and a data migration to populate the new column for historical backfills.
```

Comment:
Can we combine this entry and the one below "Added a new column to the Runs table ..." to 

- Added a new column to the BulkActions table, a new column to the Runs table, and a new BackfillTags table to improve the performance of the Runs page. To take advantage of these performance improvements, run `dagster instance migrate`. This migration involves a schema migration to add the new columns and table, and a data migration to populate the new columns for historical backfills and runs.